lib/hammer_cli_katello/activation_key.rb

```diff
@@ -219,11 +219,9 @@ class ProductContentCommand < HammerCLIKatello::ListCommand
         from :content do
           field :id, _("ID")
           field :name, _("Name")
-          field :type, _("Type")
-          field :contentUrl, _("URL")
-          field :gpgUrl, _("GPG Key")
```

Comment:
I understand the adding of the override, but not immediately why remove these at the same time?